odoo/addons/base/data/neutralize.sql

```diff
@@ -21,3 +21,7 @@ INSERT INTO ir_config_parameter (key, value)
 VALUES ('database.is_neutralized', true)
     ON CONFLICT (key) DO
        UPDATE SET value = true;
+
+-- deactivate webhooks
+UPDATE ir_act_server
```

Comment:
please make the query more specific to avoid updating potentially lots of records (e.g. `WHERE state='webhook'`)

Reply:
Done!